enterprise-suite/gotests/testenv/testenv.go

```diff
@@ -71,6 +72,7 @@ func InitEnv() {
 		cleanup(true)
 	}
 
+	additionalLbcArgs := []string{}
```

Comment:
This should work without `additionalLbcArgs`, just put what you want in `additionalArgs`.

Reply:
Hmmm.  Wouldn't that result in the `--delete-pvcs` bit getting injected after the `--` bit when invoking lbc.py in lbc.go:Install?  Around here...
```
    defaultArgs := []string{"install", "--local-chart", localChartPath,
        "--namespace", namespace,
        "--set prometheusDomain=console-backend-e2e.io",
        "--wait",
        additionalLbcArgs,
        "--", "--timeout 110"}
    fullArgs := append(defaultArgs, additionalArgs...)
    cmd := util.Cmd(lbcPath, fullArgs...)
```